reply.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef HTTP_REPLY_HPP
```

Comment:
Good call to add the reply. Makes response handling much cleaner.
Also cool that the codes for following assignments are included already.